app/controllers/stories/feeds_controller.rb

```diff
@@ -27,23 +27,49 @@ def show
     private
 
     def assign_feed_stories
-      feed = Articles::Feed.new(user: current_user, page: @page, tag: params[:tag])
       stories = if params[:timeframe].in?(Timeframer::FILTER_TIMEFRAMES)
-                  feed.top_articles_by_timeframe(timeframe: params[:timeframe])
+                  timeframe_feed
                 elsif params[:timeframe] == Timeframer::LATEST_TIMEFRAME
-                  feed.latest_feed
+                  latest_feed
                 elsif user_signed_in?
-                  ab_test_user_signed_in_feed(feed)
+                  signed_in_base_feed
                 else
-                  feed.default_home_feed(user_signed_in: user_signed_in?)
+                  signed_out_base_feed
                 end
       ArticleDecorator.decorate_collection(stories)
     end
 
-    def ab_test_user_signed_in_feed(feed)
+    def signed_in_base_feed
+      if SiteConfig.feed_strategy == "basic"
+        Feeds::Basic.new(user: current_user, page: @page, tag: params[:tag]).feed
+      else # optimized
```

Comment:
```suggestion
      else
```
I think that the method name describes this well enough that the comment is redundant here, and on line 53